Windows/MainWindow.xaml.cs

```diff
@@ -47,6 +48,8 @@ public partial class MainWindow {
 
         [CanBeNull] private CancellationTokenSource savedTimer;
 
+        public static string file_name = "none";
```

Comment:
The already exists as `targetFile`. If you need only the filename from that, `Path.GetFileName(targetFile)` or `Path.GetFileNameWithoutExtension(targetFile)`.

Reply:
Sorry, I don't remember seeing this change on the diff of the pull request...